api/src/api/agencies_v1/agency_schema.py

```diff
@@ -20,6 +20,10 @@ class AgencyListRequestSchema(Schema):
     )
 
 
+class AgencySearchRequestSchema(AgencyListRequestSchema):
+    query = fields.String()
```

Comment:
I'd recommend not deriving this from the list endpoints schema - what we want for a search request is different than what it did.

I'd recommend we follow the same pattern as the opportunity search endpoint and have filters structured the same way (using the StrSearchSchemaBuilder and such).